manifests/config.pp

```diff
@@ -61,16 +61,29 @@
     notify  => Class['jira::service'],
   }
 
-  file { "${jira::webappdir}/conf/server.xml":
-    content => template('jira/server.xml.erb'),
-    mode    => '0600',
-    require => Class['jira::install'],
-    notify  => Class['jira::service'],
-  }->
+  if ($version[0] == '7' ){
+    file { "${jira::webappdir}/conf/server.xml":
+      content => template('jira/server7.xml.erb'),
+      mode    => '0600',
+      require => Class['jira::install'],
+      notify  => Class['jira::service'],
+    }
+  }else{
+    file { "${jira::webappdir}/conf/server.xml":
+      content => template('jira/server.xml.erb'),
```

Comment:
It's possible to do it. Or maybe just instrumentate the template to be just one. But this way it's  working for me. I can adjust it on another push latter.